gravity.sh

```diff
@@ -502,6 +503,20 @@ gravity_DownloadBlocklists() {
     compression=""
     echo -e "  ${INFO} Libz compression not available\n"
   fi
+
+  # Check if etag is supported by the locally available version of curl by
+  # comparing the version string being >= 7.68.0 (released Jan 2020)
+  # https://github.com/curl/curl/pull/4543 followed by
+  # https://github.com/curl/curl/pull/4678
+  if curl -V | grep -q "curl 7\.[6-9][8-9]"; then
+    etag_support=true
+  else
+    # Check if the version is >= 8
+    if curl -V | grep -q "curl 8"; then
+      etag_support=true
+    fi
+  fi
```

Comment:
Could you save the output of `curl -V` in to a temp variable and then parse the temp variable instead of potentially calling `curl` twice in the case of `curl` version 8 or a `curl` that is less than 7.68? If most of the installs are not capable of etags then most of the installs will have two unnecessary `curl` calls. Doing a temp variable will limit that to just one `curl` call.

Reply:
Done